lib/al/Library/Camera/CameraPoser.h

```diff
@@ -14,6 +15,7 @@
 
 namespace al {
 
+struct CameraStartInfo;
```

Comment:
I'm not sure what other ones should be classes or not, due to others adding getters for variables & and then using them. Easier not to break other people's implementations by fixing this right now. I think this would be best left in another PR.

Reply:
As far as I know, all `[...]Info`s should be `struct`s. Which ones have you seen to be `class`es instead?

Reply:
There are still a few `class [...]Info`s in here.